packages/jest-matchers/src/index.js

```diff
@@ -130,6 +130,12 @@ expect.extend = (matchersObj: MatchersObject): void => {
   Object.assign(global[GLOBAL_STATE].matchers, matchersObj);
 };
 
+expect.anything = global.jasmine.anything;
+expect.any = global.jasmine.any;
+expect.objectContaining = expect.objPart = global.jasmine.objectContaining;
+expect.arrayContaining = expect.arrPart = global.jasmine.arrayContaining;
+expect.stringMatching = expect.strMatch = global.jasmine.stringMatching;
```

Comment:
There were already some aliases, like `.toBeCalled()`, that doesn't exist in raw Jasmine.
And I really like this improvement and prefer writing `toBeCalled` than `toHaveBeenCalled`.

That's why I thought, that it would be nice to declare shortcuts here as well.

@cpojer does this point make sense here?